### ISSUE TITLE
Encender should stringify objects coming out of CQL expressions

### ISSUE THREAD

**Issue by dwwinters** [ccsm-cds-tools/encender#5]
https://github.com/ccsm-cds-tools/encender/blob/main/src/main.js#L290
https://github.com/ccsm-cds-tools/encender/blob/main/src/main.js#L563

**Comment by mickohanlon23** [comment]
I believe we only want to stringify the objects coming out of CQL expressions on line #563, where the dynamicValues of the ActivityDefinitions are being evaluated, since this is where the payload of a CommunicationRequest would be formed. Stringifying the CQL output of the dynamicValues on line #293 could cause the code of a ServiceRequest to contain a string instead of a codeableConcept, as seen by the following failing test:
```
  1) CQL expression tests
       Should evaluate dynamicValue elements on an action:

      AssertionError: expected [ Array(1) ] to deeply equal [ Array(1) ]
      + expected - actual

       [
         {
      -    "code": "{\"coding\":[{\"system\":\"http://snomed.info/sct\",\"code\":\"10828004\",\"display\":\"Positive\"}],\"text\":\"I'm something\"}"
      +    "code": {
      +      "coding": [
      +        {
      +          "code": "10828004"
      +          "display": "Positive"
      +          "system": "http://snomed.info/sct"
      +        }
      +      ]
      +      "text": "I'm something"
      +    }
           "id": "54"
           "resourceType": "ServiceRequest"
           "status": "option"
           "subject": {
```

**Comment by dwwinters** [comment]
Addressed by 10cca95f97f9cfbcc3dbed7bbc102e3d8185b654.
Closing issue.
